rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4792,13 +4712,12 @@ bool opcode_019f(const ScriptArguments& args, const ScriptPlayer player, const S
 	opcode 01a0
 	@arg player 
 	@arg coord0
-	@arg coord1
+	@arg coord1 
```

Comment:
unintended?

Reply:
Pardon, file should be reformatted. Some lines are ended with space, some not.